service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/question/LinkedToHearingBlockPrePopulatorTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.fpl.service.orders.prepopulator.question;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.common.dynamic.DynamicList;
+import uk.gov.hmcts.reform.fpl.model.order.OrderQuestionBlock;
+import uk.gov.hmcts.reform.fpl.service.hearing.HearingService;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+class LinkedToHearingBlockPrePopulatorTest {
+
+    private static final List<Element<HearingBooking>> HEARINGS = List.of(
+        element(UUID.randomUUID(), mock(HearingBooking.class)));
```

Comment:
vv minor coding niggle of mine but thoughts of moving the `);` onto a new line, I think it reads better as it is how you expect braces to act and in my head brackets should act in the same way (99% of the time).

Reply:
👍